azkaban-web-server/src/main/java/cloudflow/error/CloudFlowException.java

```diff
@@ -0,0 +1,23 @@
+package cloudflow.error;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class CloudFlowException extends Exception {
```

Comment:
IMO being explicit about throwing an exception is part of the service contract, it raises awareness and forces the caller to do the appropriate handling which avoids errors at runtime that could lead to a bad user experience (users will see an error stacktrace instead of a 500 response).